apps/notifications/src/panel/templates/note-list.jsx

```diff
@@ -332,7 +332,6 @@ export class NoteList extends Component {
 		}
 
 		const classes = clsx( 'wpnc__note-list', {
-			'disable-sticky': !! window.chrome || !! window.electron, // position: sticky doesn't work in Chrome – `window.chrome` does not exist in electron
```

Comment:
This was added 6 years ago when sticky was not supported here. It seems to be supported now. 🥳 

This is what was causing the position to be different for the popover anchors. Since the presence of `position: sticky` on a parent element in firefox but not chrome made that the relative positioning different.

This also removes the need for a separate anchor placeholder element, we can just use the icon again.